test/JsonApiDotNetCoreMongoDbExampleTests/IntegrationTests/Pagination/RangeValidationWithMaximumTests.cs

```diff
@@ -0,0 +1,144 @@
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using JsonApiDotNetCore.Configuration;
+using JsonApiDotNetCore.Serialization.Objects;
+using JsonApiDotNetCoreMongoDbExample;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace JsonApiDotNetCoreMongoDbExampleTests.IntegrationTests.Pagination
+{
+    public sealed class RangeValidationWithMaximumTests : IClassFixture<IntegrationTestContext<Startup>>
```

Comment:
These tests can be removed, they verify paging query strings against configured options.